adventureGame/AdventureGame/src/Main.py

```diff
@@ -125,4 +129,63 @@ def constructDungeonRoom():
         # Print for debug.
         dungeonRoom1.printDungeonRoomInfo()
 
+def UserInput():
```

Comment:
probably need some sort of first-time instruction on how the input works. i.e do we typing action then enemy after. e.g. [walk, north] or is it the other way round.

You can do this with a flag statement.

At start of program put a inputTutorialFlag = False

if inputTutorialFlag == false:
    print("instructions")
   \#make instructions never print again.
    inputTutorialFlag = true

Reply:
See other comments